xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp

```diff
@@ -173,12 +173,7 @@ bool CLinuxRendererGL::ValidateRenderTarget()
 {
   if (!m_bValidated)
   {
-    //!@todo remove extension check?
-    if (!CServiceBroker::GetRenderSystem()->IsExtSupported("GL_ARB_texture_non_power_of_two") &&
```

Comment:
note the not operator

Reply:
oh thx...

Reply:
Is the non_power_of_two thingy still not in standard and has to be checked with ```IsExtSupported```?
If so, would it be a valid solution to change it to the following?
```
if (!CServiceBroker::GetRenderSystem()->IsExtSupported("GL_ARB_texture_non_power_of_two")
  m_textureTarget = GL_TEXTURE_RECTANGLE;
```

Reply:
no need for checking non_power_of_two. it is always supported